src/pages/image/_id.vue

```diff
@@ -33,20 +33,17 @@
       <VButton
         as="VLink"
         :href="image.foreign_landing_url"
-        class="btn-main flex-initial w-full md:w-max mb-4 md:mb-0"
+        class="btn-main flex-initial w-full md:w-max mb-4 md:mb-0 leading-[1.3]"
         size="large"
         >{{ $t('image-details.weblink') }}</VButton
       >
-      <span class="flex-1 flex flex-col justify-center">
-        <h1 class="text-base md:text-3xl font-semibold leading-[130%]">
+      <div
+        class="flex-1 flex flex-col justify-center font-semibold text-base leading-[1.3]"
+      >
+        <h1 class="md:text-3xl">
```

Comment:
```suggestion
        <h1 class="md:text-3xl font-semibold">
```

I'm noticing that the `h1` isn't inheriting the semibold `600` weight from the parent div, so it it needs this class specified.